src/bin/cli.js

```diff
@@ -172,10 +172,10 @@ const cli = async () => {
     asyncApiPath = path.resolve(options.filepath);
     scenarioPath = path.resolve(options.scenario);
   }
-  const structuredData = await verifyInputGetData(rdInterface, path.resolve(asyncApiPath),path.resolve(scenarioPath),options.basedir);
+  const dataFromParser = await verifyInputGetData(rdInterface, path.resolve(asyncApiPath),path.resolve(scenarioPath),options.basedir);
   const manager = RequestManager();
-  await manager.createReqHandler(structuredData);
-  await manager.startOperations();
+  await manager.createReqHandler(dataFromParser);
+  await manager.startScenario('SimpleGame','mqtt');
```

Comment:
🧐 

Why is this hardcoded in the CLI? 

Does this mean that you cannot use the CLI for other scenarios?

Reply:
i will change it to execute all scenarios

Reply:
i think i will limit the cli to executing all the scenarios. For more specific things sm1 would have to use the javascript api, or the desktop application. At least for now